### PR TITLE
Fixes for RHEL 7

### PR DESCRIPTION
**Pull request by mvollmer**
Followups:

- Investigate and maybe file bug about NM disconnections
- Use sudo with journalctl, maybe, and run check-services as "admin" again.
- Get SELinux and pam.d/sshd corrected

### CODE REVIEW

File: test/check-dashboard [hunk 1]
```diff
@@ -76,6 +76,14 @@ class TestDashboard(MachineCase):
         b.click('#dashboard_setup_next')
         b.wait_popdown('dashboard_setup_server_dialog')
```
Comment (stefwalter) [inline test/check-dashboard]: Should we merge #2128 instead of this?

File: test/check-networking [hunk 2]
```diff
@@ -173,4 +174,14 @@ class TestNetworking(MachineCase):
...
         b.wait_not_in_text("#networking-interfaces", "tbridge")
 
-test_main()
+if 'TEST_OS' in os.environ and os.environ['TEST_OS'] == "rhel-7":
+
+    # HACK - We lose all connectivity to the VM for about 30 seconds
+    #        after changing the config of the additional network
+    #        interfaces.
+    #
+    # XXX - file bug
```
Comment (stefwalter) [inline test/check-networking]: Sounds pretty wild. I guess lets file this bug before we merge this pull request.

File: test/check-storage [hunk 3]
```diff
@@ -560,6 +560,10 @@ class TestStorage(MachineCase):
         m = self.machine
         b = self.browser
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1187481
```
Comment (stefwalter) [inline test/check-storage]: This bug needs to be cloned for RHEL 7. There should be a 'Clone' link in the bug above.

File: test/check-dashboard (hunk 1 above)
Comment (mvollmer) [inline test/check-dashboard]: I am not sure what this refers to.

File: test/check-networking (hunk 2 above)
Comment (mvollmer) [inline test/check-networking]: Yes, true.

**Comment by mvollmer** [comment]
> Page error: TypeError: 'null' is not an object (evaluating 'e.split')

This is probably fixed by #2111.

File: test/check-dashboard (hunk 1 above)
Comment (stefwalter) [inline test/check-dashboard]: Instead of configuring /etc/pam.d/sshd

Comment (stefwalter) [inline test/check-dashboard]: Oh, and it's #2126

Comment (mvollmer) [inline test/check-dashboard]: Yes, let's do that.

File: test/check-storage (hunk 3 above)
Comment (mvollmer) [inline test/check-storage]: Done.

File: test/check-dashboard (hunk 1 above)
Comment (mvollmer) [inline test/check-dashboard]: Done.

Comment (mvollmer) [inline test/check-dashboard]: Done.

File: test/check-services [hunk 4]
```diff
@@ -65,6 +65,10 @@ Description=Test Template for %I
 ExecStart=/usr/bin/test-service %I
 """)
 
+        # Only root can read the system.journal on RHEL 7.
+        if m.os == "rhel-7":
+            b.default_user = "root"
```
Comment (stefwalter) [inline test/check-services]: Is there a bug about this? Do you know why there is a difference of behavior from Fedora?

File: test/check-networking (hunk 2 above)
Comment (mvollmer) [inline test/check-networking]: Done.

File: test/check-services (hunk 4 above)
Comment (mvollmer) [inline test/check-services]: I don't know the reason.  I'll file a bug.

Comment (mvollmer) [inline test/check-services]: https://bugzilla.redhat.com/show_bug.cgi?id=1211499:

> Access list granting read access on /var/log/journal to members of wheel group is adjusted at package install time. In RHEL there is no /var/log/journal by default so after user creates it has to setup access list on directory manually.

We create /var/log/journal explicitly, but don't adjust the permissions, apparently. Too obvious...

**Comment by dperpeet** [comment]
I haven't tested it, but the code changes appear sane

**Pull request closed without merging**